src/current/v23.1/connection-pooling.md

```diff
@@ -25,37 +25,149 @@ Each time an application reads or writes data, it will request one of the connec
 To read more about connection pooling, see our [What is Connection Pooling, and Why Should You Care](https://www.cockroachlabs.com/blog/what-is-connection-pooling/) blog post.
 {{site.data.alerts.end}}
 
-## Sizing connection pools
+## Recommended settings for connection pools
 
-Idle connections in CockroachDB do not consume many resources compared to PostgreSQL. Cockroach Labs estimates the memory overhead of idle connections in CockroachDB is 20 kB to 30 kB per connection.
+These are Cockroach Labs recommendations for settings common to most connection pooling software. If your application has multiple workers, each worker should be configured based on its own connections and workload.
 
-Creating the appropriate size pool of connections is critical to gaining maximum performance in an application. Too few connections in the pool will result in high latency as each operation waits for a connection to open up. But adding too many connections to the pool can also result in high latency as each connection thread is being run in parallel by the system. The time it takes for many threads to complete in parallel is typically higher than the time it takes a smaller number of threads to run sequentially.
+### Set the maximum lifetime of connections
+
+The maximum lifetime of a connection should be set to no less than 5 minutes and no more than 30 minutes. Applications that connect to {{ site.data.products.serverless }} clusters should set this to 30 minutes, while applications that connect to {{ site.data.products.dedicated }} and {{ site.data.products.core }} clusters should set this to 5 minutes.
+
+### Set the maximum number of open connections
+
+The maximum number of open connections should be set to a value that is greater than the maximum number of concurrent queries that your application worker expects to make. Doing so minimizes the need to establish new connections.
+
+### Set the maximum number of idle connections
+
+Set the maximum number of idle connections to the same value as the maximum number of open connections.
+
+### Set the maximum lifetime of idle connections
+
+Set the maximum lifetime of idle connections to the same value as the maximum lifetime of connections.
 
-Each processor core can only execute one thread at a time. When there are more threads than processor cores, the system will use context switching to [time-slice](https://wikipedia.org/wiki/Preemption_(computing)#Time_slice) the thread execution. For example, if you have a system with a single core and two threads, processing threads 1 and 2 in parallel results in the system context switching to pause execution of thread 1 and begin executing thread 2, and then pause execution of thread 2 to resume executing thread 1. Executing thread 1 completely and then executing thread 2 will be faster because the system doesn't need to context switch, even though thread 2 had to wait until thread 1 fully completed to begin executing.
+### Avoid spikes in new connections
```

Comment:
I think this is the "break point" on the page, where we move from specific connection pooling settings to general advice for managing the connection pool? Shouldn't we have a new sub-section to mark the boundary? The top of the page should just include the absolute bare minimum advice for customers to avoid the worst problems that happen if they don't configure connection pooling. Everything further down should be more detailed "optimization" advice that helps them squeeze out additional performance.

Reply:
+1 to the point about readability. I almost wonder if we should make these numerical recommendations formatted in a table instead of textual sections.